modelator/src/step_runner.rs

```diff
@@ -100,8 +100,11 @@ mod tests {
         let tla_config_file = "tests/integration/tla/Numbers.cfg";
         let runtime = crate::ModelatorRuntime::default();
         let mut runner = NumberSystem::default();
-        assert!(runtime
-            .run_tla_steps(tla_tests_file, tla_config_file, &mut runner)
-            .is_ok());
+        let result_test_report =
+            runtime.run_tla_steps(tla_tests_file, tla_config_file, &mut runner);
```

Comment:
you can do `assert_eq!(runtime.run_tla_steps(tla_tests_file, tla_config_file, &mut runner).map(|x| x.is_ok()), Ok(true))`

Reply:
Thanks for the tip. Your pattern didn't quite work but I found this clean alternative
```assert!(runtime
            .run_tla_steps(tla_tests_file, tla_config_file, &mut runner)
            .map_or_else(|_| false, |v| v.is_ok()),);
```